policyengine_us/variables/gov/states/il/dhs/aabd/income/earned/il_aabd_earned_income_after_exemption_person.py

```diff
@@ -0,0 +1,47 @@
+from policyengine_us.model_api import *
+
+
+class il_aabd_earned_income_after_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    definition_period = MONTH
+    label = "Illinois Aid to the Aged, Blind or Disabled (AABD) earned income after exemption per person"
+    reference = (
+        "https://www.law.cornell.edu/regulations/illinois/Ill-Admin-Code-tit-89-SS-113.120",
+    )
+    defined_for = StateCode.IL
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.il.dhs.aabd.income.exemption
+        gross_earned_income = person("il_aabd_gross_earned_income", period)
+        expense_exemption = person("il_aabd_expense_exemption_person", period)
+        adjusted_income = max_(gross_earned_income, expense_exemption)
+        # Determine demographic status
+        p_age = parameters(period).gov.states.il.dhs.aabd
+        age = person("monthly_age", period)
```

Comment:
skip the aged and disabled part, 
```suggestion
        blind = person("is_blind", period)
        exemption = where(blind, p.blind.calc(income_after_flat_exemption), p.elderly_or_disabled.calc(income_after_flat_exemption))
        return income_after_flat_exemption - exemption
```